testshared/src/main/java/org/odk/collect/testshared/RobolectricHelpers.java

```diff
@@ -99,38 +96,6 @@ public static void mountExternalStorage() {
         return null;
     }
 
```

Comment:
I think it's a good opportunity to convert this class to kotlin.

Reply:
Converted.